hidapi/hidapi.h

```diff
@@ -30,11 +30,11 @@
 #include <wchar.h>
 
 #ifdef _WIN32
-      #define HID_API_EXPORT __declspec(dllexport)
-      #define HID_API_CALL
+	  #define HID_API_EXPORT __declspec(dllexport)
```

Comment:
you've replaced some of the spaces with tabs, but not all of them

consistency (even across small block of code) is better than "some general rules"